public/manifest.json

```diff
@@ -12,11 +12,17 @@
       "128": "/images/icon.png"
     }
   },
-  "permissions": ["activeTab", "storage", "tabs", "identity", "identity.email"],
+  "icons": {
+    "16": "/images/icon.png",
+    "48": "/images/icon.png",
+    "128": "/images/icon.png"
+  },
+  "permissions": ["storage", "identity", "identity.email"],
```

Comment:
Where did the "activeTab" & "tabs" permissions go? Are they no longer necessary?

Reply:
they were likely added as part of a tutorial when implementing the microsoft oath, but they're not used anywhere in our extension (google rejects extensions with unnecessary permissions)